ephem_tod/automation/jsr223/python/community/ephem_tod/ephem_tod.py

```diff
@@ -219,8 +219,10 @@ def ephem_tod(event):
 
     # Create a timer to run this rule again a little after midnight. Work around
     # to deal with the fact that cron triggers do not appear to be workind.
-    reload_time = DateTime().now().withTime(0,2,0,0).plusDays(1)
-    log.info("Creating reload timer for {}".format(reload_time))
+    reload_time = DateTime().now().withTime(0,5,0,0)
+    if reload_time.isBefore(DateTime().now()):
```

Comment:
Just add a day to reload_time instead of creating a DateTime from scratch

```
now = DateTime().now()
reload_time = now.withTime(0,5,0,0)
if reload_time.isBefore(now):
    reload_time = reload_time.plusDays(1)
```

That shortens the code a little and makes it a little more clear, not that it's complicated code in the first place.

Reply:
Looks much better.
Added commit